docs/index.asciidoc

```diff
@@ -47,10 +47,9 @@ within 30 days of a database update.
 
 The GeoIP filter plugin can manage the database for users running the Logstash default
 distribution, or you can manage
-database updates on your own. The behavior is controlled by the `database` setting.
-When you use the default `database` setting, the auto-update feature ensures that the plugin is
+the updates on your own, becoming also responsible for maintaining compliance. The behavior is controlled by the `database` setting.
+When you use the default `database` setting, and the auto-update feature is enabled, Logstash ensures that the plugin is
```

Comment:
```suggestion
When you use the default `database` setting and the auto-update feature is enabled, Logstash ensures that the plugin is
```

Reply:
Nice clarification! Thanks for adding this detail.

Reply:
Please remember to either commit the suggestion above (removing first comma) or make the change on your own.